pages/deposit-dates.css

```diff
@@ -0,0 +1,28 @@
+.table thead > :first-child {
+  border-top-width: 2px;
+}
+
+.container {
+  display: grid;
```

Comment:
It's better to use flex, in this case and mostly. Check it for mobile.

With flex you can provide grow and shrink properties to make it resize automatically screens size independently. With greed you need a media query.

Add please 18rem minimal width to card component. It should fix the problem here if you use flex.

Reply:
Fair enough, thanks.